releasenotes/notes/remove-safe-patch-72d3e674ab9dfeda.yaml

```diff
@@ -0,0 +1,4 @@
+---
+upgrade:
+  - |
+    Remove deprecated method ``ddtrace.internal.utils.wrappers.safe_patch``.
```

Comment:
this is an internal method, no need to have a release note for it.

if it was exposed via `ddtrace.utils` then that module is gone.